projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -606,4 +611,54 @@ public static function has_file_system_write_access() {
 	public static function get_idc_container_id() {
 		return static::IDC_CONTAINER_ID;
 	}
+
+	/**
+	 * Conditionally append the red bubble notification to the "Jetpack" menu item if there are alerts to show
+	 *
+	 * @return void
+	 */
+	public static function maybe_show_red_bubble() {
+		global $menu;
+		// filters for the items in this file
+		add_filter( 'my_jetpack_red_bubble_notification_slugs', array( __CLASS__, 'alert_if_missing_site_connection' ) );
+		$red_bubble_alerts = self::get_red_bubble_alerts();
+
+		// The Jetpack menu item should be on index 3
+		if ( ! empty( $red_bubble_alerts ) && isset( $menu[3] ) && $menu[3][0] === 'Jetpack' ) {
+			// phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+			$menu[3][0] .= sprintf( ' <span class="awaiting-mod">%d</span>', count( $red_bubble_alerts ) );
```

Comment:
I don't think this will happen here, since `get_red_bubble_alerts` only returns an array, but we should check if the variable `is_countable` before calling `count` on it.

Reply:
yes, good call